livekit_sip.proto

```diff
@@ -66,20 +66,47 @@ service SIP {
 message CreateSIPTrunkRequest {
   // CIDR or IPs that traffic is accepted from
   // An empty list means all inbound traffic is accepted.
-  repeated string addresses = 1;
+  repeated string inbound_addresses = 1;
 
-  // `To` value that will be used when making a call
-  string to = 2;
+  // IP that SIP INVITE is sent too
+  string outbound_address = 2;
+
+  // Number used to make outbound calls
+  string outbound_number = 3;
 
   // Accepted `To` values. This Trunk will only accept a call made to
   // these numbers. This allows you to have distinct Trunks for different phone
   // numbers at the same provider.
-  // An empty list means all dialed numbers are accepted.
-  repeated string allowed_destinations_regex = 3;
+  repeated string inbound_numbers_regex = 4;
+
+  // Username and password used to authenticate inbound and outbound SIP invites
+  // May be empty to have no Authentication
+  string username = 5;
+  string password = 6;
```

Comment:
Not exactly a protocol issue, but are there requirements about encrypting user credentials we need to store?